Assets/BossRoom/Scripts/Editor/BakingMenu.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.Rendering;
+using UnityEngine.XR;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Scripts.Editor
+{
+    /// <summary>
+    /// This is a script that creates a menu for baking lights (and changing other lighting features) for Boss Room.
+    /// </summary>
+    public class BakingMenu : MonoBehaviour
+    {
+        static void HandleEnvLights(bool realtimeLightsEnabled, bool bakedLightsEnabled, string lightingStatus)
+        {
+            GameObject[] bakedLights = GameObject.FindGameObjectsWithTag("LightingBaked");
```

Comment:
```suggestion
            var bakedLights = GameObject.FindGameObjectsWithTag("LightingBaked");
```
Just a style guide thing. Can do this for other local variables inside this class' methods.